cmd/viam-agent-syscfg/main.go

```diff
@@ -66,15 +66,19 @@ func main() {
 	}
 
 	log.Debugf("Config: %+v", cfg)
+
+	// exact text "startup complete" is important, the parent process will watch for this line to indicate startup is successful
```

Comment:
if it's load-bearing, considering breaking it out to a constant

Reply:
Will fix this when we merge. (That's the project for this cycle.) Also, this was already that way.